src/gui/newwizard/setupwizardwindow.cpp

```diff
@@ -35,6 +36,7 @@ SetupWizardWindow::SetupWizardWindow(SettingsDialog *parent)
     , _ui(new ::Ui::SetupWizardWindow)
 {
     Utility::setModal(this);
+    setWindowFlag(Qt::WindowCloseButtonHint, false);
```

Comment:
Why do you need this? Closing the dialog via the regular exit button added by the window manager also triggers `reject`.

Apparently, on macOS one needs to combine this with `WindowSystemMenuHint`.

Reply:
This disables the X on Windows, I think we give better guidance if we force the user to use the close button?

Reply:
You mean the cancel button? I'm not sure... I like to use either option. But I'm a dev.

> Apparently, on macOS one needs to combine this with WindowSystemMenuHint.

My bad, I guess with these sheets one doesn't have to care about this anyway.